README.md

```
@@ -24,311 +24,436 @@ Download from [Releases](https://github.com/langtind/homeyctl/releases) and add
+  "actions": [
+    {"id": "homey:device:<device-id>:on", "args": {}},
+    {"id": "homey:device:<device-id>:target_temperature_set", "args": {"target_temperature": 23}}
+  ]
+}
+```
+
+### Important: Droptoken Format
+
+When referencing device capabilities in conditions, use pipe (`|`) as separator:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)